libc/src/__support/HashTable/bitmask.h

```diff
@@ -73,7 +73,7 @@ template <class BitMask> struct IteratableBitMaskAdaptor : public BitMask {
     return *this;
   }
   LIBC_INLINE IteratableBitMaskAdaptor begin() { return *this; }
-  LIBC_INLINE IteratableBitMaskAdaptor end() { return {0}; }
+  LIBC_INLINE IteratableBitMaskAdaptor end() { return {{0}}; }
```

Comment:
Is this really the best way to silence the warning? Can we not explicitly default construct the type to make it clearer what's actually being done here?